File: CS/ExpandoObject_MailMerge/frmResultingDocument.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraRichEdit.API.Native;


namespace ExpandoObject_MailMerge
{
    public partial class frmResultingDocument : Form
    {
        public frmResultingDocument()
        {
            InitializeComponent();
            //richEditControl1.Views.PrintLayoutView.ZoomFactor = 0.5f;
        }

        public Document Document { get { return richEditControl1.Document; } } 
    }
}

File: VB/ExpandoObject_MailMerge/frmResultingDocument.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraRichEdit.API.Native


Namespace ExpandoObject_MailMerge
	Partial Public Class frmResultingDocument
		Inherits Form
		Public Sub New()
			InitializeComponent()
			'richEditControl1.Views.PrintLayoutView.ZoomFactor = 0.5f;
		End Sub

		Public ReadOnly Property Document() As Document
			Get
				Return richEditControl1.Document
			End Get
		End Property
	End Class
End Namespace
